src/translate.rs

```diff
@@ -189,9 +189,7 @@ impl ToGlib for () {
     type GlibType = ();
 
     #[inline]
-    fn to_glib(&self) -> () {
```

Comment:
Why the `/*-> ()*/`?

Reply:
Because clippy not like returning unit both in declaration and body

Reply:
to_glib usually return something so I decide just comment

Reply:
Fine with me :)

Reply:
Why do we need this impl though?

Reply:
Examples compiles without it, so it will be removed